db/primary_migrate/20250416181519_create_duplicate_profile_confirmations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDuplicateProfileConfirmations < ActiveRecord::Migration[8.0]
+  def change
+    create_table :duplicate_profile_confirmations do |t|
+      t.references :profile, foreign_key: true, null: false
+      t.timestamp :confirmed_at, null: false
+      t.json :duplicate_profiles, null: false
```

Comment:
Is there an expected structure for the `json` column?

Reply:
yes its supposed to be an array of ids.

Reply:
Would an integer array `integer[]` suffice?

Reply:
It would be, but I don't see a real advantage.

The table will be updated using Ruby/Rails code and SQL queries, if any, are likely to be on the other columns.

Reply:
I think it's preferable to choose a simpler/smaller type, especially one that could be foreign keys. I also think we could also consider normalizing the table with a row for each duplicate to allow for individual updates on a given relationship for each if needed in the future?

For clarity, should the column indicate they are identifiers and be named `duplicate_profile_id(s)`?

Reply:
I considered normalizing the duplicates but didn't think it is worth it - with the current design, the user is acknowledging the set as a whole and it is fine to add a new set if the duplicates change. 

I agree that `duplicate_profile_ids` is better.

The argument of int array vs json is becoming pedantic in my opinion. I still think that JSON upholds the principle of least surprise. I am willing to go with int arrays if that will move this work along.

Reply:
Yeah, I think `integer[]` is preferable if we'll only ever be storing a list of integers.